examples/features/load_balancing/server/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary server is an example server.
+// Server shows how to respond with backend addresses to test client
```

Comment:
let's not include test. These examples are suppose to be meant for real use cases